packages/meta-extraction/src/abstractExtractionRules.js

```diff
@@ -952,15 +973,16 @@ const runAllRules = async (html, page, url) => {
   const applicableRules = rules.filter((rule) => rule.shouldApplyRule(url));
 
   for (const rule of applicableRules) {
-    const { abstract, pdf, result } = await rule.executeRule(html, page);
+    const { abstract, pdf, error } = await rule.executeRule(html, page);
 
-    if (result === 'stop') {
-      return {};
+    if (error) {
+      return {error};
     }
 
     if (abstract || pdf) {
+      let cleanedAbstract = await cleanMathjax(abstract,page);
       return {
-        abstract: cleanAbstract(abstract),
+        abstract: cleanAbstract(cleanedAbstract),
```

Comment:
Would it be better to call `cleanMathjax` inside `cleanAbstract`? To keep things cleaner (pun intended).